kuma/wiki/tests/test_views.py

```diff
@@ -339,17 +339,23 @@ def test_footer_contributors(self, get_contributors, flag_is_active):
         # just checking if the contributor link is rendered
         eq_(len(contributors.find('a')), 2)
 
-    def test_revision_view_bleached_content(self):
-        """Bug 821988: Revision content should be cleaned with bleach"""
-        d, r = doc_rev("""
-            <a href="#" onload=alert(3)>Hahaha</a>
-            <svg><svg onload=alert(3);>
-        """)
+    def test_template_revision_content(self):
+        doc = document(title='Testing Template', slug='Template:Testing', save=True)
+        r = revision(save=True, document=doc, is_approved=True)
+
         resp = self.client.get(r.get_absolute_url())
-        page = pq(resp.content)
-        ct = page.find('#wikiArticle').html()
-        ok_('<svg>' not in ct)
-        ok_('<a href="#">Hahaha</a>' in ct)
```

Comment:
I followed the code path. The `content_cleaned` is still processed with bleach a few levels into this call:
https://github.com/mozilla/kuma/blob/master/kuma/wiki/managers.py#L23-L42

I think the test and comment are still relevant. Since you're not changing this in the commit I'd leave the test as-is and keep these 2 new ones you added.